integration/kubernetes/helm-chart/alluxio/templates/master/statefulset.yaml

```diff
@@ -49,6 +49,15 @@ spec:
         release: {{ .Release.Name }}
         heritage: {{ .Release.Service }}
         role: alluxio-master
+    {{- if or .Values.master.podAnnotations .Values.metrics.enabled }}
+      annotations:
+      {{- with .Values.master.podAnnotations }}
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      {{- with .Values.metrics.podAnnotations }}
```

Comment:
It is there to add all the annotations specified in the master.podAnnotations as well as metrics.podAnnotations and as default values as `{}` provided it will not throw an error.

Reply:
Does it make sense to have metrics.podAnnotations when Prometheus is not enabled? If yes then I'm good with this part.

Reply:
I am not sure whether adding metadata via annotations will be useful for other metrics systems such as Graphite or Jmx hence keeping it when enabling metrics, given annotations will be applied to the pod. So that any other tool can use that metadata to automate the tasks.